src/hdmf/utils.py

```diff
@@ -15,6 +15,17 @@
     'data': []
 }
 
+
+def __get_docval_macros(key=None):
```

Comment:
How is this different from `get_docval_macro` on line 60?

Reply:
Better question: is this necessary, given that we already have `get_docval_macro` on line 60?

Reply:
Looking at the code, I think the function can be removed and replaced with the existing ``get_docval_macro``. Given how long this PR has been in the works, my guess is that  ``get_docval_macro`` was added at some point after I forked of things from dev. I would suggest that we simply remove  ``__get_docval_macro`` and use  ``get_docval_macro`` in its place.

Reply:
@rly I remove the ``__get_docval_macros`` function and replaced it with the existing ``get_docval_macro``.